string_assignment.js

```diff
@@ -0,0 +1,84 @@
+//1. `console.log` the first character of a string.
```

Comment:
Good demonstration of implementing string methods and properties to solve problems. How come you didn't do any of bonus questions? Those are good practice materials to practice syntax and logic.